src/decoder.js

```diff
@@ -90,17 +95,22 @@ export default class Decoder {
   decodeMethod(inputData) {
     const errorObject = { name: 'UNDECODED', params: [{ name: 'rawData', value: inputData, type: 'data' }] };
     if (inputData === '0x') return '';
+
     const methodID = inputData.slice(2, 10);
     const abiItem = this.methodIDs[methodID];
     if (abiItem) {
       const params = abiItem.inputs.map(item => item.type);
```

Comment:
I think you should also check if input is exists and has type of array to avoid the `can get data from undefiend` error.